ccd-definition/CaseField/CareSupervision/gatekeepingOrder.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "gatekeepingOrderRouter",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "GatekeepingOrderRoutes",
+    "Label": "Order type",
+    "HintText": "What do you want to do?",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "sdoDirectionCustom",
+    "Label": "Further directions",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "CustomDirection",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "sdoHearingDate",
```

Comment:
what is this doing currently - think it needs to be added at top of pages in the events - can we use existing SDO field?

Reply:
removed, hearing dates are in next PR